### PR TITLE
docs: add v17 versions support

### PR DESCRIPTION
**Pull request by JeanMeche**
* Node 16 support is dropped per angular/angular-cli#25675 
* TS 5.2 is supported per #51334
*  Node20 support per #52448

### CODE REVIEW

File: aio/content/guide/versions.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ covers [Angular versions under active support](guide/releases#actively-supported
 
 | Angular            | Node.js                              | TypeScript     | RxJS               |
 |--------------------|--------------------------------------|----------------|--------------------|
-| 16.1.x             | ^16.14.0 \|\| ^18.10.0               | >=4.9.3 <5.2.0 | ^6.5.3 \|\| ^7.4.0 |
+| 17.0.x             | ^18.13.0                             | >=4.9.3 <5.3.0 | ^6.5.3 \|\| ^7.4.0 |
```
Comment (alan-agius4) [inline aio/content/guide/versions.md:13]: I recommend that we do this in a separate PR so that we only target the main branch.

Comment (JeanMeche) [reply aio/content/guide/versions.md:13]: Should start from the 16.2.x branch for the 16.2 update ?

Comment (JeanMeche) [reply aio/content/guide/versions.md:13]: See #51448, is this what you meant ?

Comment (alan-agius4) [reply aio/content/guide/versions.md:13]: Yes, thanks

**Comment by alan-agius4** [comment]
The removal of support of Node.js 16 hasn’t actually landed yet in the framework.

**Review by alan-agius4 — APPROVED** [review]

**Review by dylhunn — APPROVED** [review]

**Comment by LayZeeDK** [comment]
issue: add `6208865206c9a9402ba21afffe982f2192a19438` to include v16.2.x version compatibility.

File: aio/content/guide/versions.md [hunk 2]
```diff
@@ -10,12 +10,11 @@ covers [Angular versions under active support](guide/releases#actively-supported
 
 | Angular            | Node.js                              | TypeScript     | RxJS               |
 | ------------------ | ------------------------------------ | -------------- | ------------------ |
+| 17.0.x             | ^18.13.0 \|\| ^20.9.0                | >=4.9.3 <5.3.0 | ^6.5.3 \|\| ^7.4.0 |
 | 16.1.x             | ^16.14.0 \|\| ^18.10.0               | >=4.9.3 <5.2.0 | ^6.5.3 \|\| ^7.4.0 |
```
Comment (LayZeeDK) [inline aio/content/guide/versions.md:14]: issue: add `6208865206c9a9402ba21afffe982f2192a19438` to include v16.2.x version compatibility.

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
Request change.

**Review by AndrewKushnir — APPROVED** [review]

**Comment by atscott** [comment]
This PR was merged into the repository by commit 289a178ef0af2c0f8ac3c41097a7bbea7985994d.

**Pull request closed without merging**